astropy/io/misc/asdf/tags/fits/fits.py

```diff
@@ -0,0 +1,86 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: utf-8 -*-
+
+import numpy as np
+
+from astropy.io import fits
+from astropy.io.misc.asdf.types import AstropyAsdfType
```

Comment:
This may be more logical as a relative import: `from ... import AstropyAsdfType`

Also the `asdf` import should be moved up, below numpy, since it is not local.

Reply:
I have no strong feelings about this, though have a slight preference for sticking to the convention we have, i.e., import relatively. As I wrote above, this makes sense especially within `io.misc.asdf` - it means we could easily move it later to `io.asdf` if that became useful (this also suggests keeping things like `astropy.io.fits` as they are...

Reply:
Sounds like a reasonable compromise. I'll post an update shortly.